sros2_cmake/CMakeLists.txt

```diff
@@ -7,10 +7,8 @@ set(LIB_INSTALL_DIR lib/)
 set(INCLUDE_INSTALL_DIR include/)
 set(SYSCONFIG_INSTALL_DIR share/${PROJECT_NAME})
 
-find_package(ament_cmake_test REQUIRED)
```

Comment:
This command sets the build `BUILD_TESTING` flag.
If we remove it, we should do something else to enable testing (I think `enable_testing()`).

Reply:
There is much more logic to it than just calling `enable_testing()`: see https://github.com/ament/ament_cmake/blob/207a07b72af1e5a780887d7c2268df57ef456af3/ament_cmake_test/ament_cmake_test-extras.cmake#L17-L37 so I don't think duplicating that logic makes sense.

Reply:
https://github.com/ros2/sros2/pull/95/commits/e4ee571a5f41fcff8c84b064c8e3d33624b44aa2 applies suggestion from https://github.com/ros2/sros2/pull/90#discussion_r264031986